README.md

```diff
@@ -539,38 +538,54 @@ options:
   -t ENTRY_TYPE, --entry-type ENTRY_TYPE
                         Name of the type (from the CDDL) to interpret the data
                         as.
+  --default-max-qty DEFAULT_MAX_QTY, --dq DEFAULT_MAX_QTY
+                        Default maximum number of repetitions when no maximum
+                        is specified. It is only relevant when handling data
+                        that will be decoded by generated code. If omitted, a
+                        large number will be used.
+  --yaml-compatibility  Whether to convert CBOR-only values to YAML-compatible
+                        ones (when converting from CBOR), or vice versa (when
+                        converting to CBOR). When this is enabled, all CBOR
+                        data is guaranteed to convert into YAML/JSON. JSON and
+                        YAML do not support all data types that CBOR/CDDL
+                        supports. bytestrings (BSTR), tags, and maps with non-
+                        text keys need special handling: All strings in
+                        JSON/YAML are text strings. If a BSTR is needed, use a
+                        dict with a single entry, with "zcbor_bstr" as the
+                        key, and the byte string (as a hex string) as the
+                        value, e.g. {"zcbor_bstr": "0123456789abcdef"}. The
+                        value can also be another type, e.g. which will be
+                        interpreted as a BSTR with the given value as contents
+                        (in cddl: 'bstr .cbor SomeType'). E.g. {"zcbor_bstr":
+                        ["first element", 2, [3]]} Dicts in JSON/YAML only
```

Comment:
```suggestion
                        interpreted as a BSTR with the given value as contents
                        (in cddl: 'bstr .cbor SomeType'), e.g. {"zcbor_bstr":
                        ["first element", 2, [3]]}. Dicts in JSON/YAML only
```

This seems more like documentation text than a helptext for a parameter. Should we consider moving it to docs and pointing to docs from the helptex?

Reply:
Yes, let's do that, removed. The README changes will happen in #253.